docs/concepts/storage/volumes.md

```diff
@@ -696,6 +697,24 @@ spec:
 
 For further detail, plese the see the [ScaleIO examples](https://github.com/kubernetes/kubernetes/tree/{{page.githubbranch}}/examples/volumes/scaleio).
 
+### local
+This volume type is alpha in 1.7.
+
+A `local` volume represents a mounted local storage device such as a disk,
+partition or directory.
+
+Local volumes can only be used as a statically created PersistentVolume source.
+
+Compared to HostPath volumes, local volumes can be used in a durable manner
+without manually scheduling pods to nodes, as the system is aware of the volume's
+node constraints.
+
+However, local volumes are still subject to the availability of the underlying
+node and are not suitable for all applications.
+
+For further detail, please see the [local volume
+user guide](https://github.com/kubernetes-incubator/external-storage/tree/master/local-volume)
```

Comment:
From what I understand, [kubernetes/examples](https://github.com/kubernetes/examples) will be where we keep some "real world" examples, which we will use in the tutorials. If there's going to be an accompanying tutorial to walk users through local persistent storage, then I'm okay w/ it going there. I think @ahmetb is managing that repo, so maybe he should chime in too.